### PR TITLE
fix(timepicker-select): add input wrapper to prevent icon from displaying incorrectly in IE11

### LINKED ISSUE #1143: Time picker select's arrow displays incorrectly on IE11
## Detailed description

> Describe in detail the issue you're having.

Issue details can be found at https://github.com/carbon-design-system/carbon/issues/5651

There is a pull request https://github.com/carbon-design-system/carbon/pull/5672 that will fix the issue for vanilla and react since the class "bx--select_arrow" is on the SVG but for angular it will make it display incorrectly in all browsers since the class will be on the container instead of the SVG.
image

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> Is this issue related to a specific component?

Time picker select

> What did you expect to happen? What happened instead? What would you like to see changed?

Arrow should display correctly on IE11. Maybe if the arrow icon uses carbon icons angular's SVG directive instead of the component. This way we can put the class `class="bx--select__arrow"` on the SVG and it should display correctly.

> What browser are you working in?

IE11

> What version of the Carbon Design System are you using?

carbon-components: 10.10.2
carbon-components-angular: 3.46.6

### PR DESCRIPTION
**Pull request by Donisius**
Closes #1143 

Still not perfect, but it looks a lot better with the wrapper. It seems like a styling issue since the problem exists in the react implementation as well.

before:
Screen Shot 2020-03-23 at 4 32 56 PM

after:
Screen Shot 2020-03-23 at 4 54 18 PM

### CODE REVIEW

**Comment by jdcai** [comment]
The pull request https://github.com/carbon-design-system/carbon/pull/5672 hasn't been merged yet but when it is merged it will fix it for vanilla and react. In angular it will cause the arrow to display incorrectly. More details in issue.

File: src/timepicker-select/timepicker-select.component.ts [hunk 1]
```diff
@@ -27,6 +28,7 @@ import { NG_VALUE_ACCESSOR } from "@angular/forms";
 				<ng-content></ng-content>
 			</select>
 			<ibm-icon-chevron-down16 *ngIf="!skeleton" class="bx--select__arrow"></ibm-icon-chevron-down16>
```
Comment (cal-smith) [inline src/timepicker-select/timepicker-select.component.ts]: Does changing to the directive form `<svg ibmIconChevronDown16></svg>` improve things at all?

Comment (Donisius) [reply src/timepicker-select/timepicker-select.component.ts]: It doesn't change anything

Comment (jdcai) [reply src/timepicker-select/timepicker-select.component.ts]: Did you try applying the styles from https://github.com/carbon-design-system/carbon/pull/5672/files after changing to the directive?

Comment (Donisius) [reply src/timepicker-select/timepicker-select.component.ts]: I haven't! I will try that out

Comment (cal-smith) [reply src/timepicker-select/timepicker-select.component.ts]: We should switch to the directive form anyway 👍

**Pull request merged**